widget/embedded/src/components/ChainsChip/ChainsChip.tsx

```diff
@@ -0,0 +1,15 @@
+import type { PropTypes } from './ChainsChip.types';
+import type { PropsWithChildren } from 'react';
+
+import React from 'react';
+
+import { Chip } from './ChainsChip.styles';
+
+export function ChainsChip(props: PropsWithChildren<PropTypes>) {
+  const { children, selected, onClick } = props;
+  return (
+    <Chip selected={selected} onClick={onClick}>
```

Comment:
When`selected` is `true`, it has a jump (because we are using `border`). For checking the jump, go to `/from-swap` then try to switch between `Select Chain` items.

Reply:
You can use `outline` instead of `border`.